rpi5.yaml

```diff
@@ -230,6 +283,10 @@ parameters:
                 gpt_type: ebd0a0a2-b9e5-4433-87c0-68b6b72699c7
                 type: vfat
                 size: 128 MiB
+                items:
+                  "dom0/z_sync.bin": "%{ZEPHYR_DOMU1_DIR}/%{ZEPHYR_DOMU1_BUILD_DIR}/zephyr/zephyr.bin"
+                  "dom0/z_blinky.bin": "%{ZEPHYR_DOMU2_DIR}/%{ZEPHYR_DOMU2_BUILD_DIR}/zephyr/zephyr.bin"
+                  "dom0/helloworld_xen-arm64": "domu_unikraft/helloworld_xen-arm64"
```

Comment:
please drop dom0 folder.

Reply:
Nope. It's current default for dom0 app. It's kept the same also to avoid confusion if switching bootfs/dom0fs

Reply:
ok